Client/core/Graphics/CRenderItemManager.cpp

```diff
@@ -803,6 +805,11 @@ void CRenderItemManager::GetDxStatus(SDxStatus& outStatus)
         outStatus.settings.bGrassEffect = false;
     }
 
+    if (outStatus.settings.iFXQuality < 2)
+    {
+        outStatus.settings.bDynamicPedShadows = false;
+    }
+
```

Comment:
According to your code, a user could run `dynamic_ped_shadows=1` to enable the ped shadows and see them, but in dx-status you would still see it disabled.

Reply:
It's not gonna work if FX quality is lower than "high". GTA does a check somewhere.